packages/core/build/config.js

```diff
@@ -183,6 +183,7 @@ const htmlInjectConfig = () => ({
             path: 'manifest.json',
             attributes: {
                 rel: 'manifest',
+                crossorigin: 'use-credentials',
```

Comment:
I've chosen to use credentials to help reconcile possible cross-origin restrictions that might lead to the manifest fetching failure.